api-metastore/src/test/java/org/zalando/nakadi/controller/EventTypeControllerTest.java

```diff
@@ -566,32 +566,6 @@ public void whenDeleteNoneExistingEventTypeThen404() throws Exception {
                 .andExpect(content().contentType("application/problem+json"));
     }
 
-    @Test
-    public void whenDeleteEventTypeThatHasSubscriptionsThenConflict() throws Exception {
```

Comment:
See this test instead: https://github.com/zalando/nakadi/blob/master/acceptance-test/src/acceptance-test/java/org/zalando/nakadi/webservice/hila/SubscriptionAT.java#L419